.gitignore

```diff
@@ -127,3 +127,5 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+.vscode/
```

Comment:
this belongs in [global gitignore](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c)

Reply:
yes, I will remove it in the other pull request